app/tests/util/webhooks.py

```diff
@@ -0,0 +1,55 @@
+import time
+import os
+from typing import List, Literal
+
+from httpx import AsyncClient
+import httpx
+
+from app.tests.util.string import get_wallet_id_from_JWT
+
+WEBHOOKS_URL = os.getenv("WEBHOOKS_URL", "http://localhost:3010")
```

Comment:
We have a constants file for tests that contain env variables for tests: https://github.com/didx-xyz/aries-cloudapi-python/blob/development/app/tests/util/constants.py

Reply:
Ah yeah true. Sorry I went the old way of doing that. You;r absolutely right